src/cycle-manager/cycle-manager.service.ts

```diff
@@ -16,79 +16,110 @@ export class CycleManagerService {
   ) {}
   game: any = new Game(PLAYER_CONFIG, INITIAL_BET);
 
+  nextRound() {
+    this.game.endRound();
+    this.game.startRound();
+  }
+
   async startRound(roomId) {
     const playersInRoom = await this.getPlayersInRoom(roomId);
     if (playersInRoom.length <= 1) {
       console.log('Wait for at least one more player.');
       return [];
```

Comment:
in future you could throw error here to handle it gracefully but for now this is fine